var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -127,6 +127,8 @@ def cmake_enable_option(spec_requiremement):
                                                              "+tests"]]
         if "+mpi" in self.spec:
             args.append("-DNRN_ENABLE_MPI_DYNAMIC=ON")
+        else:
```

Comment:
do we want mpi enabled or disabled if there is no reference to mpi in the spec?

Reply:
As mpi is variant, there is always `mpi` in the spec. It has value either `+mpi` or `~mpi`.